scripts/setupEnzyme.ts

```diff
@@ -1,4 +1,18 @@
 import Adapter from 'enzyme-adapter-react-16';
 import Enzyme from 'enzyme';
 
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
```

Comment:
1. Let's sort them.
2. We don't use that for the tests, therefore lets put no-op functions here. `jest.fn` record all calls and will increase memory usage.